openmls/src/group/core_group/mod.rs

```diff
@@ -186,17 +194,36 @@ impl CoreGroupBuilder {
     /// [`OpenMlsCryptoProvider`].
     pub(crate) fn build(
         self,
+        credential_bundle: &CredentialBundle,
+        life_time: LifetimeExtension,
```

Comment:
Passing in a LifetimeExtension could probably be an optional build step `with_lifetime_extension`, where by default we just use the default one.